CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Tableau Connector SDK Changelog
 
+## Unreleased
```

Comment:
There's some clean-up to occur in this file as well. This can be a follow-up PR.

Reply:
Will adress both in a follow up PR once the 2021.2 changes are merged into master